classes/robot/crawler.php

```diff
@@ -535,7 +535,9 @@ public function get_old_queue_size() {
     }
 
     /**
-     * Pops an item off the queue and processes it
+     * Crawl as many URL's as we can in the time limit
+     * This function will run in parallel with the adhoc_crawl_task
```

Comment:
The adhoc task is not super important, it's more correct to say it will run in parallel with itself. ie you could have the cron running as well as run it from the cli

Reply:
I have updated this